applications/sel_phase_optimizer/sel_phase_linac.py

```diff
@@ -59,6 +62,24 @@ def q_waveform(self):
             self._q_waveform_pv = PV(self.pv_addr("CTRL:QWF"))
         return self._q_waveform_pv.get()
 
+    @property
+    def fit_chisquare_pv_obj(self) -> PV:
```

Comment:
It would be nice to have tests for these modeled after `test_sel_poff_pv`

Reply:
done.